qt/scientific_interfaces/Indirect/JumpFit.ui

```diff
@@ -223,6 +223,9 @@
              </item>
              <item>
               <widget class="QComboBox" name="cbPlotType">
+               <property name="enabled">
```

Comment:
Can you explain this change? Is it intentional to disable the plot types for JumpFit?

Reply:
Yes, I set the combobox to be disabled initially when you first open the MSD Fit tab because there is no output data. This combobox is then enabled later on after a 'Run' when there is output data to be plotted.

Reply:
I see. Could you also update the documentation page for the MSD Fit interface?

Reply:
Absolutely, is it the MSD Fit Tab image which you'd like to be updated in the documentation?

Reply:
I think the image is generated automatically, but if not, then yes it would be nice to keep the screenshot up to date. Plus a sentence about the new feature would be fine.

Reply:
It looks like this new feature has already been included in the documentation for this tab - even though it wasn't available for this specific tab before. http://docs.mantidproject.org/nightly/interfaces/Indirect%20Data%20Analysis.html#output